src/app-layout/__tests__/split-panel.test.tsx

```diff
@@ -124,7 +128,7 @@ for (const theme of ['refresh', 'classic']) {
             onSplitPanelPreferencesChange={noop}
           />
         );
-        act(() => wrapper.findSplitPanel()!.findOpenButton()!.click());
```

Comment:
Just dropping `act` everywhere because they are useless. `.click()` already does `act` when needed.

Reply:
because we already do it inside the test-util: https://github.com/cloudscape-design/test-utils/blob/595baac332382d22b87ad210f26010a02cbc3f67/src/core/dom.ts#L65-L67

No need to duplicate. Also, it is for testing development experience of our consumers, see AWSUI-22860. 

Every time you feel like you need to do act in our tests, it is probably a sign that we should release a test-util giving it to our consumers automatically.

Eventually I expect to remove all manual `act` from our tests, little-by-little, like in this PR. If you can clean our code too when doing your changes, it would be great